docs/send-data/opentelemetry-collector/remote-management/source-templates/linux.md

```diff
@@ -0,0 +1,89 @@
+---
+id: linux
+title: Linux Source Template
+sidebar_label: Linux
+description: Learn about the Sumo Logic Linux source template for OpenTelemetry.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+<img src={useBaseUrl('img/integrations/hosts-operating-systems/linux-transparent.png')} alt="Thumbnail icon" width="45"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="30"/>
+
+Linux source template creates an OpenTelemetry configuration that can be pushed to a remotely managed OpenTelemetry collector (abbreviated as otelcol). By creating this source template and pushing the config to the appropriate OpenTelemetry agent you can ensure collection of linux logs and host metrics of linux to Sumo Logic.
+		
+## Fields creation in Sumo Logic for Linux
+
+If not already present, the following [Fields](/docs/manage/fields/) are created as part of source template creation.
+
+- **`sumo.datasource`**. Fixed value of **linux**.
+- **`deployment.environment`** User configured field at the time of collector installation. This identifies the environment where the linux system resides. For example: dev, prod or qa.
+- **`host.group`**. This is a collector level field and is user configured (at the time of collector installation). Through this, the linux host group is identified.
+- **`host.name`**. This is tagged through the resourcedetection processor. It holds the value of the host name where the OTel collector is installed.
```

Comment:
```suggestion
- **`host.name`**. This is tagged through the resource detection processor. It holds the value of the host name where the OTel collector is installed.
```

Reply:
The name of the processor itself is resource detection https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor that is why I have kept this way